### PR TITLE
adding private preview docs for private clusters

### PR DESCRIPTION
**Pull request by sakthi-vetrivel**
@jimzim @ehashman

### CODE REVIEW

**Comment by ehashman** [comment]
@sakthi-vetrivel I don't think this repo is the right place for this documentation. In order to be rendered on the website, you'll need to submit a PR to https://github.com/MicrosoftDocs/azure-docs-pr/

See e.g. https://github.com/MicrosoftDocs/azure-docs-pr/pull/98777

**Comment by sakthi-vetrivel** [comment]
@ehashman Since this documentation is for a private preview feature, I'm planning on landing the documentation here first, and then migrating to Microsoft docs when we reach public preview.

File: private-preview-docs/private-cluster.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Create an ARO 3.11 Private Cluster
+
+[!IMPORTANT]
+Please note that ARO private clusters are currently only available in private preview in East US. Private preview acceptance is by invitation only. Please be sure to register your subscription before attempting to enable this feature.
```
Comment (jimzim) [inline private-preview-docs/private-cluster.md]: Needs to be EastUS2

File: private-preview-docs/private-cluster.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+## Sample Configuration Scripts
+
+### Environment
+
+Fill in the environment variables below as appropriate, using your cluster name, subscription ID, preferred location, etc.
+
+```
+export CLUSTER_NAME=
+export LOCATION=westcentralus
```
Comment (jimzim) [inline private-preview-docs/private-cluster.md]: Change westcentralus to eastus2

**Pull request merged**